addons/website_sale/views/delivery_form_templates.xml

```diff
@@ -91,7 +91,8 @@
                 </span>
                 <span
                     name="o_pickup_location_selector"
-                    class="btn btn-primary fa fa-pencil ms-2 p-1"
+                    t-attf-class="btn btn-primary fa fa-pencil ms-2 p-1
+                                  {{'' if len(dm.warehouse_ids) > 1 else 'd-none'}}"
```

Comment:
warehouse field is define in website_sale_collect accessing it here will cause traceback. No? :thinking: